### PR TITLE
SEO-174730 JavaScript Description too long

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK ,
I have made changes in the description section, please review.
Regards,
Scholar.

### CODE REVIEW

File: js/Button/Repeat-Button.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: Repeat-Button | Syncfusion
-description: repeat button
+title: Repeat-Button in JavaScript Button | Syncfusion
```
Comment (MallikaRK) [inline js/Button/Repeat-Button.md:3]: don't use repeated keyword in single sentence
,
Repeat-Button in JavaScript | Syncfusion - change it.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: js/Button/Repeat-Button.md (hunk 1 above)
Comment (Scholar-jaruha) [reply js/Button/Repeat-Button.md]: resolved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**